examples/ChatQnA/ChatQnA_Guide.rst

```diff
@@ -142,20 +132,85 @@ See :doc:`/GenAIExamples/supported_examples`
 Architecture
 ************
 
-TODO: Includes microservice level graphics.
+The ChatQnA architecture is displayed below:
+![alt text](image.png)
+
+.. figure:: /GenAIExamples/ChatQnA/assets/img/chatqna_flow_chart.png
+   :alt: ChatQnA Architecture Diagram
+
+.. code-block:: python
```

Comment:
```suggestion
.. mermaid::
```
And then everything that's the mermaid graph description below needs to be indented three spaces